build.rs

```diff
@@ -281,108 +281,92 @@ fn make_zlib(compiler: &cc::Tool, src_dir: &path::Path, _: &path::Path) {
     emit_rerun_directives_for_contents(&src_dir);
 }
 
-fn make_elfutils(compiler: &cc::Tool, src_dir: &path::Path, out_dir: &path::Path) {
+fn make_elfutils(compiler: &cc::Tool, src_dir: &path::Path, _: &path::Path) {
     // lock README such that if two crates are trying to compile
     // this at the same time (eg libbpf-rs libbpf-cargo)
     // they wont trample each other
     let file = std::fs::File::open(src_dir.join("elfutils/README")).unwrap();
     let _lock = fcntl::Flock::lock(file, fcntl::FlockArg::LockExclusive).unwrap();
 
-    let flags = compiler
-        .cflags_env()
-        .into_string()
-        .expect("failed to get cflags");
-    let mut cflags: String = flags
-        .split_whitespace()
-        .filter_map(|arg| {
-            if arg != "-static" {
-                // compilation fails with -static flag
-                Some(format!(" {arg}"))
-            } else {
-                None
-            }
-        })
-        .collect();
+    let project_dir = src_dir.join("elfutils");
+    let project = project_dir.to_str().unwrap();
 
-    #[cfg(target_arch = "aarch64")]
-    cflags.push_str(" -Wno-error=stringop-overflow");
-    cflags.push_str(&format!(" -I{}/zlib/", src_dir.display()));
+    let host = if build_android() {
+        format!("")
```

Comment:
Why is there no target triple available?

Reply:
> Why is there no target triple available?

When I compile for android locally, an error will be reported, so I leave it blank.

Reply:
That doesn't really answer or explain anything. `aarch64-linux-android` is a valid triple for Android, among [others](https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-2-without-host-tools). We should probably understand what the issue is instead of working around it from the get-go.